addon/components/au-radio.js

```diff
@@ -0,0 +1,29 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class AuRadioComponent extends Component {
+  get checked() {
+    const { inGroup, checked, groupValue, value } = this.args;
+    return inGroup ? value === groupValue : checked;
+  }
+
+  get value() {
+    const { value } = this.args;
+    return value === undefined ? undefined : String(value);
+  }
```

Comment:
I know it's implemented like this in the velvet version, but we don't need this here since we don't use TS / Glint. I would just pass through the argument to the attribute directly.